src/main/antlr4/org/folio/circulation/rules/CirculationRules.g4

```diff
@@ -147,7 +147,10 @@ criterium : CRITERIUM_LETTER
 
 all : 'all';
 
-fallbackpolicy : 'fallback-policy' policies NEWLINE
+fallbackpolicy : 'fallback-policy'
+                ( policies NEWLINE
+                | NEWLINE { notifyErrorListeners("Policy missing."); }
```

Comment:
Do we only need to add this to a fallback policy?

Reply:
no, but for other cases I added handling in Text2Drools.java